kava-protocol-home.js

```diff
@@ -425,10 +410,10 @@ const setRewardsApyDisplayValues = async (denoms, siteData, cssIds) => {
   const denomConversions = siteData['denomConversions']
 
   for (const denom of denoms) {
-
+    if (denom === 'usdx') continue; 
```

Comment:
Nice - no need to calculate a reward apy for a denom that won't ever be used as a cdpType 👍